README.md

```diff
@@ -91,28 +91,34 @@ For reference, here is a truth table for the expression A && B. You can fill ou
 
 | A     | B     | A && B |  A ll B   |
 | ----- | ----- | ------ | ---------- |
-| true  | true  | true   |      |
-| false | true  | false  |      |
-| true  | false | false  |      |
-| false | false | false  |      |
+| true  | true  | true   |   true   |
+| false | true  | false  |    true  |
+| true  | false | false  |   true   |
+| false | false | false  |    false  |
 
 ## Problem Four
 
 Create a truth table for the expression !A && !B.
 
 For reference, here is a truth table for the expression A && !B. You can fill out the last column. Don't worry about keeping the spacing exact.
 
-| A     | B     | !B    | A && !B | !A && !B |
-| ----- | ----- | ----- | ------- | -------- |
-| true  | true  | false | false   |          |
-| false | true  | false | false   |          |
-| true  | false | true  | true    |          |
-| false | false | true  | false   |          |
+| A     | B     | !B    | A && !B | !A && !B  |
```

Comment:
Missing last column. Please revisit and make sure it makes sense. When it doubt try the code in node. !true && !true.